src/main/scala/tech/cryptonomic/conseil/tezos/PlatformDiscoveryTypes.scala

```diff
@@ -45,20 +46,20 @@ object PlatformDiscoveryTypes {
   }
 
   /**
-    * Function maps slick column type as a string to DataType
-    * @param tpe type as a string from slick TableQuery
+    * Function maps slick jdbc column type to DataType
+    * @param tpe type from slick TableQuery
     * @return one of DataType enums
     */
-  def mapType(tpe: String): DataType = {
-    val optionRegex = "Option\\[([A-Za-z0-9']+)\\]".r
+  def mapType(tpe: Type): DataType = {
```

Comment:
This is a great improvement

Will it create problems if we need to use any other db than postgres? Say, H2 for testing?

Reply:
if we introduce more complex types it may, but for the generated ones I don't think so

Reply:
Are you sure that the postgres-profile-specific import won't create issues with other slick profiles? I was referring to that essentially

Reply:
there are tests in place which are working fine - the whole schema is tested just because of this case. I think we should probably think abut using tests with the same database as we are using in production i.e. `postgresql-embedded` or with some docker containers.

Reply:
As long as tests work it's all fine.

About using embedded pg, it might make sense if it's fast. Using docker is ok for me, but only in integration tests. In my experience it gets burdensome on a daily working cycle. Unit tests are supposed to be slick